newrelic/hooks/mlmodel_sklearn.py

```diff
@@ -66,6 +139,33 @@ def _instrument_sklearn_models(module, model_classes):
             _nr_instrument_model(module, model_cls)
 
 
+def _bind_scorer(y_true, y_pred, *args, **kwargs):
+    return y_true, y_pred, args, kwargs
+
+
+def wrap_metric_scorer(wrapped, instance, args, kwargs):
+    transaction = current_transaction()
+    # If there is no transaction, do not wrap anything.
+    if not transaction:
+        return wrapped(*args, **kwargs)
+
+    score = wrapped(*args, **kwargs)
+
+    y_true, y_pred, args, kwargs = _bind_scorer(*args, **kwargs)
+    model_name = "Unknown"
+    if hasattr(y_pred, "_nr_wrapped_model_name"):
+        model_name = y_pred._nr_wrapped_model_name
+    # Attribute values must be int, float, str, or boolean. If it's not one of these
+    # types and an iterable add the values as separate attributes.
+    if not isinstance(score, (str, int, float, bool)):
+        if hasattr(score, "__iter__"):
+            for i, s in enumerate(score):
+                transaction._add_agent_attribute("%s.%s[%s]" % (model_name, wrapped.__name__, i), s)
```

Comment:
Branch coverage says this isn't covered at all, we should ensure this is tested.